src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/BlobProcessorTask.java

```diff
@@ -236,13 +237,15 @@ private ZipFileProcessingResult processZipFileContent(
 
             InputEnvelope envelope = envelopeProcessor.parseEnvelope(result.getMetadata(), zipFilename);
 
+            EnvelopeProcessor.assertEnvelopeContainsOcrDataForNewApplication(envelope);
```

Comment:
perhaps this could be called only for SSCS

Reply:
or `assertEnvelopeContainsOcrDataIfRequired` or sth like that

Reply:
I couldn't think of any other better name, so I renamed it as `assertEnvelopeContainsOcrDataIfRequired`

Reply:
I meant 'named', though :)